pkg/scheduler/core/generic_scheduler.go

```diff
@@ -201,7 +201,7 @@ func (g *genericScheduler) Schedule(pod *v1.Pod, nodeLister algorithm.NodeLister
 		return result, err
 	}
 
-	trace.Step("Computing predicates")
+	trace.Step("About to compute predicates")
```

Comment:
It would make more sense if this log message described the work that happend before it, instead of the work that is about to happen.

Reply:
@misterikkit Fixed.